app/scripts/controllers/dashboard.ts

```diff
@@ -11,6 +11,7 @@ module labsFrontendApp {
         releases: any;
         feature: any;
         message: any;
+        feature_sections: any;
```

Comment:
Yeah, I might change that. This was a feature which I accidentally removed earlier and now just put it back quickly :'D